main/core/src/mill/eval/Evaluator.scala

```diff
@@ -637,7 +708,8 @@ class Evaluator @deprecated(message = "Use apply instead", since = "mill 0.10.1"
     workerCache,
     env,
     failFast,
-    threadCount
+    threadCount,
+    this.importTree
```

Comment:
I think this copy method does not fulfil the purpose of bin compat, as it already contains a changed signature.

Reply:
It doesn't. I changed only the implementation here to copy maintaining the old `importTree`